updatehub/src/states/mod.rs

```diff
@@ -307,7 +307,7 @@ impl StateMachine {
 /// use updatehub;
 ///
 /// updatehub::logger::init(slog::Level::Info);
-/// let settings = updatehub::Settings::load()?;
+/// let settings = updatehub::Settings::load(std::path::PathBuf::from("/etc/updatehub.conf"))?;
```

Comment:
This will need to be updated once `Settings::load` have the change I just requested.